components/sidebar/sidebar_category_list/sidebar_category_list.tsx

```diff
@@ -330,6 +349,62 @@ export default class SidebarCategoryList extends React.PureComponent<Props, Stat
         this.updateUnreadIndicators();
     }, 100);
 
+    onBeforeCapture = (before: BeforeCapture) => {
+        // // Ensure no channels are animating
+        this.channelRefs.forEach((ref) => ref.classList.remove('animating'));
+
+        // TODO: Refactor into better code
```

Comment:
Do you have concrete plans for refactoring this, or can we remove this comment? We shouldn't be leaving TODOs around without an accompanying ticket.

Reply:
I don't, it's just not code I'm happy with. I'd like to look into a better way to control the scrolling, any suggestions?

Reply:
No, but we shouldn't be leaving comments around for stuff that we don't have anything actionable for improving or else we'll just be cluttering the code.

Reply:
I'll remove the TODO then and we can revisit later on.